pkg/streamingpromql/operators/test_operator.go

```diff
@@ -39,6 +39,10 @@ func (t *TestOperator) SeriesMetadata(_ context.Context) ([]types.SeriesMetadata
 	seriesPool = seriesPool[:len(t.Series)]
```

Comment:
[nit, realise this was here before] `seriesPool` doesn't make a lot of sense in this context - how about just `series` or `metadata` or something like that?

Reply:
Updated in 541526b08f8f373d450fce12822a89267fab9bf0